.circleci/config.yml

```diff
@@ -1,15 +1,23 @@
 version: 2
 workflows:
   version: 2
-  test:
+  # test:
+  #   jobs:
+  #     - contract_cw1_subkeys
+  #     - contract_cw1_whitelist
+  #     - contract_cw20_base
+  #     - contract_cw20_escrow
+  #     - package_cw1
+  #     - package_cw20
+  #     - lint
+  deploy:
     jobs:
-      - contract_cw1_subkeys
-      - contract_cw1_whitelist
-      - contract_cw20_base
-      - contract_cw20_escrow
-      - package_cw1
-      - package_cw20
-      - lint
+      - build_and_upload_contracts
+          # filters:
+          #   tags:
+          #     only: /^v[0-9]+\.[0-9]+\.[0-9]+.*/
```

Comment:
Yeah, and other code to enable once it works. Got it.

Reply:
Yes, to activate the filter, you need to add a `:` after `build_and_upload_contracts` as well